test/services/pin-merging/hooks/prepare-activity-log.test.js

```diff
@@ -0,0 +1,86 @@
+// Test helper functions
+const assertTestEnv = require('../../../test_helper').assertTestEnv;
+const expect = require('../../../test_helper').expect;
+const loadFixture = require('../../../test_helper').loadFixture;
+const stub = require('../../../test_helper').stub;
+
+// Models
+const PinModel = require('../../../../src/services/pin/pin-model');
+
+// Fixtures
+const pins = require('../../../fixtures/pins');
+const ASSIGNED_PIN_ID = require('../../../fixtures/constants').ASSIGNED_PIN_ID;
+const ASSIGNED_PIN_DETAIL = require('../../../fixtures/constants').ASSIGNED_PIN_DETAIL;
+const VERIFIED_PIN_ID = require('../../../fixtures/constants').VERIFIED_PIN_ID;
```

Comment:
ok. Renamed to PIN_ASSIGNED_ID, PIN_ASSIGNED_DETAIL, DEPARTMENT_GENERAL_ID, etc.